### ISSUE TITLE
Regarding transcript_bed and peak_bed

### ISSUE THREAD

**Issue by desperado39** [mhorlacher/sc2rbpmap#1]
Can you please tell, what is the transcript_bed file and peak_bed file. i didn't understand the difference between them ?

**Comment by mhorlacher** [comment]
Hi,

The transcript BED file is taken from reference annotations of RNA transcripts, e.g. mRNAs. The peak BED file specifies regions that were identified as RBP-binding by a peak caller from CLIP-seq data. `generate_background_no-peak.py` uses both to sample a set of regions within transcripts that do not overlap with peaks, which can then be used as negatives.

**Comment by desperado39** [comment]
Thank you for you reply.
So, you also use ```generate_background_no-peak.py``` for making the input sequence uniform length (ref from paper i.e. 400 nt).
or you use some other method to create input file from binding regions sequences of different length.

**Comment by mhorlacher** [comment]
Yes, we generate sequences of uniform length, which can be controlled with the `--window-size` argument (default is 400, i.e. same as in the paper).

**Comment by desperado39** [comment]
Thank you for reply.
So, making the input sequence uniform length is it because of you want to increase the coverage of the genome and increase its accuracy or its random. Can you please elaborate this. This will be very helpful.

**Comment by mhorlacher** [comment]
When training our models, we used a fixed-length input of 400nt, centered around CLIP-seq peaks. Then, when generating negative samples, we ensure that they have same length. When using different input sizes for the positive samples, the size of the negative samples (here referred to as "background") need to be adjusted accordingly. Hope that helped!

**Comment by desperado39** [comment]
Hello,
Can you please tell me what is the sequence of preprocessing steps that you follow to get the positive and negative sets for model building. In this paper it is not very much clear how you do that.
I follwed these steps:
1. Intersect the data from gtf file converted in bed file of the genome(release 35).
2. getting the 5-prime end through the code you provided.
3. making the sequence of length 400
4. both replicates interesected for getting the required overlap of 0.75 
But by following these steps the number sequences found to be very less (around 30000).
please tell me the preprocessing steps in more detail. I am very obliged to you.
